files/ru/learn/javascript/first_steps/what_went_wrong/index.md

```diff
@@ -66,12 +66,12 @@ original_slug: Learn/JavaScript/Первые_шаги/Что_пошло_не_т
 
     > **Примечание:** [`Null`](/en-US/docs/Glossary/Null) — это специальное значение, которое означает "ничего" или "не значение". Поэтому `lowOrHi` был объявлен и инициализирован без значения — у него нет типа или значения.
 
-    > **Примечание:** Эта ошибка не появилась, как только страница была загружена, потому что эта ошибка произошла внутри функции (внутри `checkGuess() { ... }` block). Об этом вы узнаете более подробно в нашей более поздней статье о функциях, код внутри функций выполняется в отдельной области для кода внешних функций. В этом случае код не был запущен, и ошибка не была брошена до тех пор, пока функция `checkGuess()` не была запущена строкой 86.
+    > **Примечание:** Эта ошибка не появилась, как только страница была загружена, потому что эта ошибка произошла внутри функции (внутри `checkGuess() { ... }` блока). Об этом вы узнаете более подробно в нашей более поздней статье о функциях, код внутри функций выполняется в отдельной области для кода внешних функций. В этом случае код не был запущен, и ошибка не была брошена до тех пор, пока функция `checkGuess()` не была запущена строкой 86.
 
 4. Посмотрите на строку 78, и вы увидите следующий код:
 
     ```js
-    lowOrHi.textContent = «Последнее предположение было слишком высоко!»;
+    lowOrHi.textContent = «Последнее предположение было слишком высоким!»;
```

Comment:
```suggestion
    lowOrHi.textContent = 'Last guess was too high!';
```

Reply:
[Source document](https://github.com/mdn/learning-area/blob/main/javascript/introduction-to-js-1/troubleshooting/number-game-errors.html) all in English, it will be better do not translate the code examples